sql/core/src/test/scala/org/apache/spark/sql/parquet/ParquetSchemaSuite.scala

```diff
@@ -226,22 +226,54 @@ class ParquetSchemaSuite extends FunSuite with ParquetTest {
           StructField("UPPERCase", IntegerType, nullable = true))))
     }
 
-    // Conflicting field count
+    // Metastore schema contains additional non-nullable fields.
```

Comment:
This shouldn't happen as Hive always assumes all fields are nullable. But leaving this test here should still be OK.